mmv1/products/bigquerybiglake/Database.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 Google Inc.
```

Comment:
You need a product.yaml file in this folder to trigger the generation of the resource.

Have you been able to generate this yourself locally?

Reply:
The product.yaml is in go/magic-modules/pull/8751

lmk if any ideas on how to make this review process smoother.

Reply:
Hmmmm, yeah that's a little tricky. If the resources were independent I would recommend including the product.yaml in both PRs. But since this resource requires a catalog in the URL it only makes sense in the context of having the catalog resource created. 

Maybe include both the product.yaml and catalog resource in this PR and I can focus only on the database resource?

Reply:
We submitted, which added Catalog and product.yaml.

go/magic-modules/pull/8751